models/AppModel.js

```diff
@@ -0,0 +1,16 @@
+//Create an individual video model
+
+var AppModel = Backbone.Model.extend({
+
+  defaults: function() {
+    return {
+      videos: new VideoCollection(),
+      current_video: null,
```

Comment:
If current video stores the video model for the current video, what do these other fields hold?